### ISSUE TITLE
Browser preview of publicly shared MP3 file without cover not possible

### ISSUE THREAD

**Issue by daemonixx** [owncloud/music#596]
### Steps to reproduce
1. Activate/install Music App.
2. Upload MP3 file without cover information / album art into ownCloud .
3. Share that file using a public link.
4. Open public link in a browser.

### Expected behaviour
When opening the link it should be possible to play the file without downloading it by clicking on the cover image (publicpreview).

### Actual behaviour
When opening the link no cover image is shown. Playing the file without downloading is 
  - not possible in Firefox 57 (no area to click is available) 
  - possible in Internet Explorer 11 (place holder for missing cover image is shown which is also clickable)

OS: Windows 7

### Server configuration
**Operating system:** Raspbian GNU/Linux 9.1 (Debian Stretch)
**Web server:** Apache 2.4.25
**Database:** MariaDB 10.1.23
**PHP version:** PHP 7.0.19
**ownCloud version:** ownCloud 10.0.3
**List of activated apps:** 
```
  - calendar: 1.5.4
  - comments: 0.3.0
  - configreport: 0.1.1
  - contacts: 1.5.3
  - dav: 0.3.0
  - diagnostics: 0.1.2
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_clipboard: 0.6.2
  - files_external: 0.7.1
  - files_pdfviewer: 0.8.2
  - files_sharing: 0.10.1
  - files_texteditor: 2.2.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 16.0.2
  - impersonate: 0.1.1
  - market: 0.2.3
  - music: 0.5.4
  - notifications: 0.3.1
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.1
  - updatenotification: 0.2.1
```
### Proposed solution
Implementing a dummy place holder graphic for MP3 files that do not have cover information / album art.

#### P.S.: 
The advice how to contribute given in https://github.com/owncloud/music/blob/master/CONTRIBUTING.md provides a dead link (404) to the template:
https://raw.githubusercontent.com/owncloud/core/master/issue_template.md
The correct link should be:
https://raw.githubusercontent.com/owncloud/core/master/.github/issue_template.md

**Comment by paulijar** [comment]
This looks like a bug in the ownCloud core. The placeholder for the album art is visible on ownCloud 8.2, owncloud 9.1, and Nextcloud 12, but not in ownCloud 10. Furhermore, ownCloud 10 shows the placeholder for .ogg files but not for .mp3.

**Comment by daemonixx** [comment]
Thanks for the quick feedback! I didn't know that this was already implemented in other versions of ownCloud/Nextcloud. :-)

However, I tried it in demo instances for Nextcloud 11, 12 and 13 (e.g. https://help.nextcloud.com/t/demo-instances-of-9-10-11-and-12-now-live/8709) but there it didn't work for me with MP3 files without album art. Testing it with OGG file on Nextcloud 11 and 12 worked (placeholder is visible).

**Comment by paulijar** [comment]
You are right about Nextcloud and I was mistaken. I guess I was testing only OGG files on NC, before realizing that there is a difference between the behavior of OGG and MP3. I also double-checked that my statement about oC8.2 and oC9.1 was correct: there you can see the image placeholder on MP3 files with no cover.

The deal with the older ownCloud versions is that you cannot get the new Music app versions (including the "embedded" file/share player) from the app store but you still can install it manually and it works.

As there are so many cloud versions suffering from this bug, maybe I need to hack something together in the Music app. I don't quite like the idea as that kind of solutions may easily blow up with the future cloud versions, but it might still be the least bad alternative.

**Comment by paulijar** [comment]
The workaround for this issue has now been released in the Music app v0.5.5.
